<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-QNE8PHJ" failures="1" tests="4" name="Default test" time="2.286" errors="0" timestamp="2022-08-27T12:17:00 IST">
  <testcase classname="POM.MethodOfTestNG" name="test2" time="0.021"/>
  <testcase classname="POM.MethodOfTestNG" name="test2" time="0.004"/>
  <testcase classname="POM.MethodOfTestNG" name="test1" time="2.052">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method POM.MethodOfTestNG.test1() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method POM.MethodOfTestNG.test1() didn't finish within the time-out 2000
at java.base@18.0.2/java.lang.Thread.sleep(Native Method)
at app//POM.MethodOfTestNG.test1(MethodOfTestNG.java:9)
at java.base@18.0.2/java.lang.invoke.LambdaForm$DMH/0x0000000800d00400.invokeVirtual(LambdaForm$DMH)
at java.base@18.0.2/java.lang.invoke.LambdaForm$MH/0x0000000800d00c00.invoke(LambdaForm$MH)
at java.base@18.0.2/java.lang.invoke.Invokers$Holder.invokeExact_MT(Invokers$Holder)
at java.base@18.0.2/jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(DirectMethodHandleAccessor.java:154)
at java.base@18.0.2/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base@18.0.2/java.lang.reflect.Method.invoke(Method.java:577)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:44)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:72)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:10)
at java.base@18.0.2/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@18.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@18.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@18.0.2/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="POM.MethodOfTestNG" name="test4" time="0.001">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase classname="POM.MethodOfTestNG" name="test" time="0.0">
    <ignored/>
  </testcase> <!-- test -->
</testsuite> <!-- Default test -->
